Cargo.toml

```diff
@@ -2,21 +2,20 @@
 name = "dazeus-plugin-karma"
 version = "0.1.0"
 authors = ["Ruben Nijveld <ruben@gewooniets.nl>"]
+edition = "2018"
 
-[dependencies.dazeus]
-version = "0.4"
-
-[dependencies.docopt]
-version = "0.6"
+[dependencies]
+chrono = { version = "0.4.19", features = ["serde"] }
+dazeus = "0.4"
+docopt = "1.1.0"
+env_logger = "0.8.1"
+log = "0.4.11"
+nom = "6.0.0"
+serde = { version = "1.0.117", features = ["derive"] }
+serde_json = "1.0.59"
```

Comment:
These version code specify all dependencies that are "compatible" with the version specified, i.e. all versions with the same major version.

---

See also <https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html>, in particular:

> The string "0.1.12" is a semver version requirement. Since this string does not have any operators in it, it is interpreted the same way as if we had specified "^0.1.12", which is called a caret requirement.
> [...]
> Caret requirements allow SemVer compatible updates to a specified version. An update is allowed if the new version number does not modify the left-most non-zero digit in the major, minor, patch grouping.

Reply:
Ack, I always forget if `0.1` => `0.1.x` or `0.1.1` => `0.1.x`